# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/x11/xfree86/macintosh-xkb-us-de-ibook.patch
# ROCK Linux is Copyright (C) 1998 - 2004 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

Otherwise all the third and fourth level symbols are not reachable on
iBooks :-(((

  Rene Rebe <rene@rocklinux.org>

--- xc/programs/xkbcomp/symbols/macintosh/us_ibook.orig	1970-01-01 01:00:00.000000000 +0100
+++ xc/programs/xkbcomp/symbols/macintosh/us_ibook	2003-09-04 01:15:45.000000000 +0200
@@ -0,0 +1,23 @@
+// $XConsortium: de /main/4 1996/08/31 12:19:20 kaleb $
+// $XFree86: xc/programs/xkbcomp/symbols/macintosh/de,v 1.1 2000/09/19 12:46:25 eich Exp $
+
+// rene@rocklinux.org - make the iBook useable
+
+default partial alphanumeric_keys 
+xkb_symbols "basic" {
+
+    include "macintosh/us(basic)"
+
+    // Describes the differences between a very simple US/ASCII
+    // keyboard and a very simple iBook keybaord
+
+    // Alphanumeric section
+    name[Group1]= "US/ASCII";
+    key <BKSP> {	[ BackSpace,    BackSpace		],
+			[ Delete				]	};
+    // begin modifier mappings
+    key <LMTA> {	[ Alt_L,	Meta_L	]	};
+    key <RMTA> {	[ Alt_R,	Meta_R	]	};
+
+};
+
--- xc/programs/xkbcomp/symbols/macintosh/de_ibook.orig	1970-01-01 01:00:00.000000000 +0100
+++ xc/programs/xkbcomp/symbols/macintosh/de_ibook	2003-09-04 01:15:56.000000000 +0200
@@ -0,0 +1,25 @@
+// $XConsortium: de /main/4 1996/08/31 12:19:20 kaleb $
+// $XFree86: xc/programs/xkbcomp/symbols/macintosh/de,v 1.1 2000/09/19 12:46:25 eich Exp $
+
+// rene@rocklinux.org - make the iBook useable
+
+default partial alphanumeric_keys 
+xkb_symbols "basic" {
+    include "macintosh/de(basic)"
+
+    // Describes the differences between a very simple US/ASCII
+    // keyboard and a very simple iBook keybaord
+
+    // Alphanumeric section
+    name[Group1]= "German";
+    key <BKSP> {	[ BackSpace,    BackSpace		],
+			[ Delete				]	};
+
+    // begin modifier mappings
+    key <LMTA> {	[ Alt_L,	Meta_L	]	};
+    key <RMTA> {	[ Alt_R,	Meta_R	]	};
+
+    key <LALT> {	[ Mode_switch,	Multi_key	]	};
+
+};
+
--- xc/programs/xkbcomp/symbols/macintosh/Imakefile.orig	2003-12-03 16:48:53.000000000 +0100
+++ xc/programs/xkbcomp/symbols/macintosh/Imakefile	2003-12-03 16:49:14.000000000 +0100
@@ -3,7 +3,7 @@
 XCOMM
 XCOMM $XFree86: xc/programs/xkbcomp/symbols/macintosh/Imakefile,v 1.3 2000/10/02 02:32:17 tsi Exp $
 
-     DATAFILES = de de_CH dk es fi fr fr_CH gb it nl no pt se us
+     DATAFILES = de de_CH de_ibook dk es fi fr fr_CH gb it nl no pt se us us_ibook
 
 all::
 
